site/src/pages/tutorials/thrift/blog/implement-create.mdx

```diff
@@ -0,0 +1,157 @@
+  ```java filename=BlogServiceImpl.java
+  @Override
+  public void createBlogPost(CreateBlogPostRequest request, AsyncMethodCallback<BlogPost> resultHandler)
+          throws TException {
+    ...
+    blogPosts.put(id, blogPost);
+    final BlogPost stored = blogPost;
+    resultHandler.onComplete(stored);
+  }
+  ```
+
+## 2. Implement client-side
+
+In the `BlogClient` class, add a method to send a request to create a blog post.
+
+```java filename=BlogClient.java
+import example.armeria.blog.thrift.BlogPost;
+import example.armeria.blog.thrift.CreateBlogPostRequest;
+...
+BlogPost createBlogPost(String title, String content) throws TException {
+  final CreateBlogPostRequest request =
+          new CreateBlogPostRequest().setTitle(title)
+                                     .setContent(content);
+  return blogService.createBlogPost(request);
+}
+```
+
+## 3. Register a ServerExtension
+
+In the previous test code, we've connected directly to the server.
+This time, let's use Armeria's <type://ServerExtension> instead.
+This approach simplifies set-up and tear-down of a server.
+
+In the `BlogServiceTest` class, register a <type://ServerExtension> as follows.
+Note that the service instance is added to the configuration.
+
+```java filename=BlogServiceTest.java
+import org.junit.jupiter.api.extension.RegisterExtension;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.thrift.THttpService;
+import com.linecorp.armeria.testing.junit5.server.ServerExtension;
+...
+@RegisterExtension
+static final ServerExtension server = new ServerExtension() {
+  @Override
+  protected void configure(ServerBuilder sb) throws Exception {
+    sb.service("/thrift",
+               THttpService.builder()
+                           // Add the service to the configuration
+                           .addService(new BlogServiceImpl())
+                           .build());
+  }
+};
+```
+
+## 4. Test creating a blog post
+
+Let's test if we can create a blog post.
+
+1. In the `BlogServiceTest` class, add a test method as follows.
+  ```java filename=BlogServiceTest.java
+  import static org.assertj.core.api.Assertions.assertThat;
+
+  import org.apache.thrift.TException;
+  import org.junit.jupiter.api.Order;
+  import org.junit.jupiter.api.Test;
+
+  import example.armeria.blog.thrift.BlogPost;
+
+  ...
+  @Test
+  void createBlogPost() throws TException {
+    final BlogClient client = new BlogClient(server.httpUri(), "/thrift");
+    final BlogPost response = client.createBlogPost("My first blog", "Hello Armeria!");
+    assertThat(response.getId()).isGreaterThanOrEqualTo(0);
+    assertThat(response.getTitle()).isEqualTo("My first blog");
+    assertThat(response.getContent()).isEqualTo("Hello Armeria!");
+    System.out.println(response);
+  }
+  ```
+2. Run the test case on your IDE or using Gradle.
+  ```bash
+  ./gradlew test
```

Comment:
I think the previous `hello()` test method may prevent the test from running correctly.

What do you think of either: 
1. suggesting that users remove the test or add a `@Disabled` annotation
2. modifying the uri so that `server.httpUri()` is used instead of the `"http://127.0.0.1:8080"`

Reply:
> 1. suggesting that users remove the test or add a @Disabled annotation

Done here: [c4358d1](https://github.com/line/armeria/pull/4865/commits/c4358d187bdde4dfb8c2e9423c0cd2019630d69e)